pkg/blockatlas/tx.go

```diff
@@ -53,6 +53,10 @@ const TxPerPage = 25
 // TxPage is a page of transactions
 type TxPage []Tx
 
+func (txs TxPage) Len() int           { return len(txs) }
+func (txs TxPage) Swap(i, j int)      { txs[i], txs[j] = txs[j], txs[i] }
```

Comment:
is this some type of protocol to support `Sort` ?

Reply:
yes!!!

Reply:
Good to know, thanks